<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	border-image:url(:/assets/ui.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>706</x>
      <y>240</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color:transparent;
    font-size:20px;
    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif '等线';
    color: #ffffff;
    outline: 0px;
    border: none;
}

QListWidget::item {
    padding-top: 10px;
    padding-bottom: 10px;
}

QListWidget::item:hover{
    background-color:3a4047 
	color:#080F0;
}
QListWidget::item:selected:!active{
    background-color:3a4047 
	color:#080F0;
}

QListWidget::item:selected:active
{
	background-color:#40444b;
	color:#050505;
}</string>
    </property>
    <item>
     <property name="text">
      <string>fuck</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>you</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>511</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#playButton{
	background-color:transparent;
	background-image:url(:/assets/pause_idle.png);
	outline:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>525</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#nextButton{
	background-color:transparent;
	border-image:url(:/assets/next_idle.png);
	outline:0px
}
#nextButton:pressed{
	background-color:transparent;
	border-image:url(:/assets/next_selected.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>524</y>
      <width>48</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#prevButton{
	background-color:transparent;
	border-image:url(:/assets/prev_idle.png);
	outline:0px;
}
#prevButton:pressed{
	background-color:transparent;
	border-image:url(:/assets/prev_selected.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>566</y>
      <width>400</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
   background-color: #11A4F0;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="songLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>518</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#songLabel{
	font-family:'consolas','等线';
	font-size:25px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>歌名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>528</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#uploadButton{
	background-color:transparent;
	border-image:url(:/assets/upload_idle.png);
	outline:0px
}
#uploadButton:pressed{
	background-color:transparent;
	border-image:url(:/assets/upload_selected.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="baseLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>539</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-family:'consolas','等线';
font-size:25px;
color:white;
text-align:center
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>539</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color:transparent;
	border-image:url(:/assets/plus.png);
	outline:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>539</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color:transparent;
	border-image:url(:/assets/minus.png);
	outline:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="songLabel_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>537</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-family:'consolas','等线';
	font-size:25px;
	color:white;</string>
    </property>
    <property name="text">
     <string>偏移</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
